src/server.ts

```diff
@@ -0,0 +1,58 @@
+import * as express from 'express'
+import * as ctx from './ApplicationContext'
+import * as session from 'express-session'
+import * as sessionFileStore from 'session-file-store'
+import * as cookieParser from 'cookie-parser'
+import {Auth} from './identity/auth'
+import * as log4js from 'log4js'
+import * as config from './config'
+
+const logger = log4js.getLogger('server')
+const expressNunjucks = require('express-nunjucks')
+
+const appRoot = require('app-root-path')
+
+const {PORT = 3005} = process.env
+const app = express()
+const FileStore = sessionFileStore(session)
```

Comment:
File storage can't be used in clustered environments (yeah I know it's in UI at the mo) consider changing to something like [https://github.com/mozilla/node-client-sessions](url)